src/findlib/topfind.ml.in

```diff
@@ -8,7 +8,8 @@ let predicates = ref ("toploop" :: Findlib.recorded_predicates());;
      Findlib, hence we maintain our own list
    *)
 
-let directories = ref [ Findlib.ocaml_stdlib() ];;
+let directories = 
+  Lazy.from_fun (fun () -> ref [ Findlib.ocaml_stdlib() ]);;
```

Comment:
[The manual](https://v2.ocaml.org/api/Lazy.html#VALfrom_fun) expressly recommends not doing this 🙂

How about:
```suggestion
let ocaml_stdlib = Lazy.from_fun Findlib.ocaml_stdlib;;
let directories = ref [ ] ;;
```
and then...